utils/das_data_sorter.py

```diff
@@ -1,6 +1,79 @@
 import pandas as pd 
 import argparse 
 from math import ceil
+from numpy import median, mean
+
+# averages data within the seconds or minutes, according to new_time array
+def avg_Data(array):
+    avg_array = []
+
+    for millisecond_indices in new_time:
+        avg = mean(array[millisecond_indices])
+        avg = round(avg, ndigits=2)
+        avg_array.append(avg)
+
+    return avg_array
+
+# 3 point moving mean smoothing
+def mean_smooth(data):
+    new_data = []
+
+    for i in range(1,len(data)-1):
+        first = data[i-1]
+        second = data[i]
+        third = data[i+1]
+
+        new_data_point = mean([first, second, third])
+        new_data_point = round(new_data_point, ndigits=2)
+        new_data.append(new_data_point)
+    
+    return new_data
+
+# 3 point moving median smoothing
+def median_smooth(data):
+    new_data = []
+
+    # iterates through each data point and its adjacents (excluding first and last) and finds the median 
+    for i in range(1, len(data)-1):
+        first = data[i-1]
+        second = data[i]
+        third = data[i+1]
+
+        new_data_point = median([first, second, third])
+        new_data.append(new_data_point)
+    
+    return new_data
+
+# smoothing technique function caller
+def smooth_data(data, technique):
+    if technique == "mean":
+        # when input is "--smooth mean"
+        final_data = {
```

Comment:
I think I would have preferred if you had iterated through the keys and values of the `data` dictionary and then added it to the `final_data` dictionary. This is because the keys for both dictionaries are the same.

Reply:
How do I specify the names for the keys (which represent each column of data without hardcoding it?